app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/CustomPlaybackTransportControlGlue.java

```diff
@@ -197,6 +273,15 @@ public void onCustomActionClicked(Action action, View view) {
         }
     }
 
+    public void setEndTime() {
```

Comment:
I think the implementation of this method could be added to the existing `updatePlayState` method.

Reply:
I was thinking to leave it as is so if we want to call it somewhere else we don't need duplicate code.

Reply:
Actually do you mean the function call or the contents of the function itself? Do we want the setEndTime function to be private? For now I will make it private and create the handler inside the Glue class so the TextView can't be modified externally.